shdoc

```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/gawk -f
-
+##!/bin/sh
```

Comment:
I do not have access to the Mac environment, but shouldn't `#!/usr/bin/env -S gawk -f` be sufficient here?

Reply:
See https://unix.stackexchange.com/questions/97141/distributing-a-script-should-i-use-bin-gawk-or-usr-bin-gawk-for-shebang